pyproject.toml

```diff
@@ -88,11 +88,11 @@ disable_error_code = ["no-untyped-def"]
 
 [tool.semantic_release]
 branch = "main"
+build_command = "python -m build ."
 version_variables = [
     "gitlab/_version.py:__version__",
 ]
-commit_subject = "chore: release v{version}"
-commit_message = ""
+commit_message = "chore: release v{version}"
```

Comment:
`commit_subject` is gone from v8 so it was failing. and we have to explicitly add `build_command` now otherwise it won't automatically build the wheel / source.